tests/e2e/zookeeper_test.go

```diff
@@ -25,11 +25,43 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+// createZookeeperClusterIfDoesNotExist creates a zookeeper cluster if
+// there isn't a preexisting one
+func createZookeeperClusterIfDoesNotExist(kubectlOptions *k8s.KubectlOptions, zkClusterReplicaCount int) {
```

Comment:
I think we should move the zookeeperCluster related tests in separate file like you did for kafkaCluster